Shopist/Makefile

```diff
@@ -0,0 +1,11 @@
+all: setup
+
+define SECRETS
+RUM_APPLICATION_ID=${RUM_APPLICATION_ID}\n
+DATADOG_CLIENT_TOKEN=${DATADOG_CLIENT_TOKEN}\n
+SHOPIST_BASE_URL=${SHOPIST_BASE_URL}\n
+endef
+export SECRETS
+
+setup:
+	@echo $$SECRETS > ../xcconfigs/Shopist.local.xcconfig;
```

Comment:
Which branch / git refs will we use to seed the Demo data? How will we ensure that we don't break it with recent development? I.e. if the recent change to `master` breaks the data consistency, how we will notice that we're breaking the data in Demo environment?

Reply:
![Screenshot 2020-08-18 at 15 56 27](https://user-images.githubusercontent.com/1417500/90521941-66c29580-e16b-11ea-9332-cf2533c5c57b.png)

1. `shopist_demo`: Shown in Demo org dashboards, to be triggered regularly
2. `shopist_org2`: Shown in Org2 dashboards, to be triggered manually for verification

Reply:
Actually on Android the Shopist and library development are done in two separate repository, to ensure we don't couple the Shopist app with the lib implementation. That way we also simulate exactly how an application would work "in the wild" : 
 - fetch the sdk library from the appropriate dependency management system
 - only use released versions
 - no possibility to use shorthand tricks or hacks

Reply:
@xgouchet the idea for iOS is to run `shopist_demo` only from the last release tag - this should ensure the same level of safety 👌.